pkgs/by-name/go/gow/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gow";
+  version = "unstable-2023-08-26";
```

Typo, October is the 10th month. 
```suggestion
  version = "unstable-2023-10-26";
```